modules/api/src/main/java/org/eclipse/xpanse/api/AdminServicesApi.java

```diff
@@ -89,10 +94,39 @@ private List<BackendSystemStatus> checkHealthOfBackendSystem() {
                     backendSystemStatuses.add(databaseStatus);
                 }
             }
+            if (Objects.equals(BackendSystemType.TERRAFORM_BOOT, type)) {
+                BackendSystemStatus terraformBootStatus = getTerraformBootStatus();
+                processShownFields(terraformBootStatus);
+                backendSystemStatuses.add(terraformBootStatus);
+            }
         }
         return backendSystemStatuses;
     }
 
+    private boolean isTerraformBootApiAccessible() {
+        try {
+            TerraformBootSystemStatus terraformBootSystemStatus = terraformApi.healthCheck();
+            return terraformBootSystemStatus.getHealthStatus()
+                    .equals(TerraformBootSystemStatus.HealthStatusEnum.OK);
+        } catch (Exception e) {
+            return false;
+        }
+    }
+
+    private BackendSystemStatus getTerraformBootStatus() {
+        BackendSystemStatus terraformBootStatus = new BackendSystemStatus();
+        terraformBootStatus.setBackendSystemType(BackendSystemType.TERRAFORM_BOOT);
+        terraformBootStatus.setName(BackendSystemType.TERRAFORM_BOOT.toValue());
+        if (isTerraformBootApiAccessible()) {
+            terraformBootStatus.setHealthStatus(HealthStatus.OK);
+            ApiClient apiClient = new ApiClient();
+            terraformBootStatus.setEndpoint(apiClient.getBasePath());
```

Comment:
we must also customize the base path config. 

The basepath must be read from spring-properties and then added to the ApiClient. 

The generated class has the default url and this must not be directly used.